src/utils/businessHours.ts

```diff
@@ -25,13 +25,21 @@ interface BusinessHourWindow {
   end: moment.Moment;
 }
 
-export function calculateTimeToRespondBy(
-  numDays: number,
-  productArea: string,
-  repo: string,
-  org: string,
-  testTimestamp?: string
-): string {
+interface CalculateTimetoRespondByParams {
+  numDays: number;
+  productArea: string;
+  repo: string;
+  org: string;
+  testTimestamp?: string;
+}
+
+export function calculateTimeToRespondBy({
+  numDays,
+  productArea,
+  repo,
+  org,
+  testTimestamp,
```

Comment:
nit: Is this an argument that we only pass in for testing? I wonder if there's another way to do this, a la pythong's freezgun.